utils/superset/db_name_change.py

```diff
@@ -0,0 +1,52 @@
+from io import IOBase
+import os
+import json
+import argparse
+import logging
+from pyexpat import ErrorString
+import sys
+
+
+DATABASE_NAME = "template_database_name"
```

Comment:
Could this be `TEMPLATE_DATABASE_NAME` so it stands out in the JSON?

Reply:
This is not required now with python dict string replacement.